docs/classes/request.md

```diff
@@ -2,86 +2,128 @@
 title: Request
 permalink: /classes/request
 ---
+# Request
+
+Class **`Request`** extends [`Message`](message.md)
+
+The `Request` class represents an HTTP request. It's used in [request handlers](requesthandler.md) and [`middleware`](middleware.md).
 
 * Table of Contents
 {:toc}
 
-The `Request` interface generally finds its only use in responder callables (or [`Websocket::onOpen()`](websocket.html#onopenint-clientid-handshakedata)). [`Middleware`s](middleware.md) do never see the `Request`; the `StandardRequest` class is supposed to be a simple request API reading from and manipulating [`InternalRequest`](internalrequest.md) under the hood.
+## `Constructor`
```

Comment:
`Constructor` shouldn't be in code tags IMO.

Reply:
Fixed.